Scripts/Structs/After/RemoveMonstersAndPlacemonLimits.lua

```diff
@@ -174,7 +174,7 @@ if mmver > 6 then
 
 		end
 		mem.IgnoreProtection(false)
-		ChangeGameArray("MonstersTxt", NewStartPos, NewMonCount)
+		ChangeGameArray("MonstersTxt", NewStartPos, NewMonCount + 1)
```

Comment:
`if mmver == 7 then` (line 41) section calls SimpleReplacePtr 4 times within those 40 lines of code. Maybe it's a good idea to write a comment here with some short explanation?